script/database_exists

```diff
@@ -13,11 +13,10 @@ if ($ARGV[0])
 {
     my $db = Databases->get_connection(uc($ARGV[0]));
 
-    my $sys = Databases->get_connection('SYSTEM');
-    my $sql = Sql->new( $sys->conn );
+    my $sql = Sql->new( $db->conn );
 
     my $so = $sql->select_single_value(
-        "SELECT TRUE FROM pg_database WHERE datname = ?",
+        "SELECT TRUE FROM information_schema.information_schema_catalog_name WHERE catalog_name = ?",
```

Comment:
I didn't catch why this was changed. Is it because `pg_database` requires a superuser connection while `information_schema.information_schema_catalog_name` doesn't? (I can probably research the answer, but can you quickly mention it in the commit message too? :))

Reply:
Yes, it is exactly because it no longer uses `postgres` user.

Reply:
Note pending rebase at https://github.com/metabrainz/musicbrainz-server/pull/1337/commits/4944c13d8bf980515888b9a4a63c952357475d58